index.js

```diff
@@ -20,4 +22,156 @@ closeHam.addEventListener('click', () => {
+    featureImg: 'assets/images/snapshoot-portfolio.png',
+    technology: ['html', 'css', 'javascript'],
+    platform: 'CANOPY',
+    stack: 'Back End Dev',
+    year: 2015,
+    liveLink: '#',
+    linkSource: '#',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    desc: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    featureImg: 'assets/images/Snapshoot-portfolio-4.png',
+    technology: ['html', 'Ruby on Rails', 'css', 'javascript'],
+    platform: 'FACEBOOK',
+    stack: 'Full Stack Dev',
+    year: 2015,
+    liveLink: '#',
+    linkSource: '#',
+  },
+  {
+    id: 3,
+    name: 'Facebook 360',
+    desc: "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    featureImg: 'assets/images/Snapshoot-portfolio-1.png',
+    technology: ['html', 'Ruby on Rails', 'css', 'javascript'],
+    platform: 'FACEBOOK',
+    stack: 'Full Stack Dev',
+    year: 2015,
+    liveLink: '#',
+    linkSource: '#',
+  },
+  {
+    id: 4,
+    name: 'Uber Navigation',
+    desc: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    featureImg: 'assets/images/Snapshoot-portfolio-2.png',
+    technology: ['html', 'Ruby on Rails', 'css', 'javascript'],
+    platform: 'Uber',
+    stack: 'Lead Developer',
+    year: 2018,
+    liveLink: '#',
+    linkSource: '#',
+  },
+
+  {
+    id: 5,
+    name: 'Portfolio Website',
+    desc: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    featureImg: 'assets/images/Snapshoot-portfolio-3.png',
+    technology: ['html', 'Ruby on Rails', 'css', 'javascript'],
+    platform: 'Instagram',
+    stack: 'Full Stack Dev',
+    year: 2016,
+    liveLink: '#',
+    linkSource: '#',
+  },
+
+];
+
+function generateProduct() {
+  projectsSide.innerHTML = projects.map((product) => `
+<div class="projects">
+<img src=${product.featureImg} alt="project 1" class="project-image">
+<div>
+    <div class="project-details">
+        <h2 class="project-tile">${product.name}</h2>
+        <div class="stack-year">
+            <h3 class="title">${product.platform}</h3>
+            <img src="assets/icons/Counter.png" alt="counter">
+            <h4 class="lang-year">${product.stack}</h4>
+            <img src="assets/icons/Counter.png" alt="counter">
+            <h4 class="lang-year">${product.year}</h4>
+        </div>
+    </div>
+    <p class="detail-text">${product.desc}</p>
+    <ul class="btns">
+    ${product.technology.map((x) => `<li class="btn">${x}</li>`).join('')}
+    </ul>
+    <button class="detail-btn see-p" id=${product.id}>See project</button>
+</div>
+</div>
+`).join('');
+}
+// const generateProduct = () => ();
```

Comment:
- [ ] Kindly consider removing this commented code to make your codebase clean and follow the best practices.